packages/react-utils/src/helpers/utils.tsx

```diff
@@ -1,8 +1,17 @@
+import { Meta } from '@smile-cdr/fhirts/dist/FHIR-R4/classes/meta';
 import { Dictionary } from '@onaio/utils';
 import type { i18n as i18nInstance } from 'i18next';
 
+/**
+ * From T, convert a set of keys to optional, that are in the union K.
+ */
 export type Optional<T, K extends keyof T> = Omit<T, K> & Partial<T>;
 
+/**
+ * From T, convert a set of keys to required, that are in the union K.
+ */
+export type Require<T, K extends keyof T> = Omit<T, K> & Required<Pick<T, K>>;
```

Comment:
curious: whats the purpose of `Omit<T, K>`, does not seem like it changes anything, but I could be wrong.

Reply:
lets suppose you have keys **_K_** in Interface **_T_** and you want them to be changed Only them to Required you cant do it because required will make all of them to **_Required_** I.E opposite of **_Partial_** but alternatively you can combine **_Pick_** to pick those keys make them required and also remove them from the original **_T_** and combine both of them and what you will get is the only selected keys converted to required not all